docs/api-reference/next/image.md

```diff
@@ -14,182 +14,222 @@ description: Enable Image Optimization with the built-in Image component.
+| [`src`](#src)                             | `src="/profile.png"`                 | String          | Yes      |
+| [`width`](#width)                         | `width={500}`                        | Integer (px)    | Yes      |
+| [`height`](#height)                       | `height={500}`                       | Integer (px)    | Yes      |
+| [`alt`](#alt)                             | `alt="Picture of the author"`        | String          | Yes      |
+| [`loader`](#loader)                       | `loader={imageLoader}`               | Function        | -        |
+| [`fill`](#fill)                           | `fill={true}`                        | Boolean         | -        |
+| [`sizes`](#sizes)                         | `sizes="(max-width: 768px) 100vw"`   | String          | -        |
+| [`quality`](#quality)                     | `quality={80}`                       | Integer (1-100) | -        |
+| [`priority`](#priority)                   | `priority={true}`                    | Boolean         | -        |
+| [`placeholder`](#placeholder)             | `placeholder="blur"`                 | String          | -        |
+| [`style`](#style)                         | `style={{objectFit: "contain"}}`     | Object          | -        |
+| [`onLoadingComplete`](#onLoadingComplete) | `onLoadingComplete={img => done())}` | Function        | -        |
+| [`onLoad`](#onLoad)                       | `onLoad={event => done())}`          | Function        | -        |
+| [`onError`](#onError)                     | `onError(event => fail()}`           | Function        | -        |
+| [`loading`](#loading)                     | `loading="lazy"`                     | String          | -        |
+| [`blurDataURL`](#blurDataURL)             | `blurDataURL="data:image/jpeg..."`   | String          | -        |
+
+### Required Props
+
+The Image Component requires the following properties: `src`, `width`, `height`, and `alt`.
+
+```jsx:app/page.js
+import Image from 'next/image';
+
+export default function Page() {
+  return (
+    <div>
+      <Image
+        src="/profile.png"
+        width={500}
+        height={500}
+        alt="Picture of the author"
+      />
+    </div>
+  );
+}
+```
 
-### src
+#### `src`
 
 Must be one of the following:
 
-1. A [statically imported](/docs/basic-features/image-optimization.md#local-images) image file, or
-2. A path string. This can be either an absolute external URL,
-   or an internal path depending on the [loader](#loader) prop.
+1. A [locally imported](/docs/basic-features/image-optimization.md#local-images) image file. Or,
+2. A path string. This can be either an external URL or an internal path.
 
-When using an external URL, you must add it to [remotePatterns](#remote-patterns) in `next.config.js`.
+> **Good to know**: When using an external URL, you must add it to [remotePatterns](##remotepatterns) option in `next.config.js`.
```

Comment:
We'll need to update any links on other pages referencing this. Also check any error messages and redirect those.

Reply:
Did some find all searching, good call.